### PR TITLE
mkdir in convert_lit_checkpoint

### PR DESCRIPTION
**Pull request by rasbt**
This was missing a `mkdir out/hf-tinyllama` to make the code fully reproducible

### CODE REVIEW

File: tutorials/convert_lit_models.md [hunk 1]
```diff
@@ -123,6 +123,8 @@ python scripts/merge_lora.py \
 5. Convert the finetuning model back into a HF format:
 
 ```bash
+mkdir out/hf-tinyllama
```
Comment (carmocca) [inline tutorials/convert_lit_models.md:126]: I would put the mkdir inside `convert_lit_checkpoint`

**Comment by rasbt** [comment]
Is this `os.makedirs` approach ok, or should we do something fancier?

File: scripts/convert_lit_checkpoint.py [hunk 2]
```diff
@@ -247,6 +248,8 @@ def check_conversion_supported(lit_weights: Dict[str, torch.Tensor]) -> None:
 def convert_lit_checkpoint(checkpoint_path: Path, output_path: Path, config_path: Path) -> None:
     config = Config.from_json(config_path)
 
+    os.makedirs(os.path.dirname(output_path), exist_ok=True)
```
Comment (carmocca) [inline scripts/convert_lit_checkpoint.py:251]:
```suggestion
    output_path.parent.mkdir(parents=True, exist_ok=True)
```

**Review by carmocca — APPROVED** [review]

File: scripts/convert_lit_checkpoint.py [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # Copyright Lightning AI. Licensed under the Apache License 2.0, see LICENSE file.
 
 import gc
+import os
```
Comment (carmocca) [inline scripts/convert_lit_checkpoint.py:4]:
```suggestion
```

File: scripts/convert_lit_checkpoint.py (hunk 2 above)
Comment (rasbt) [reply scripts/convert_lit_checkpoint.py:250]: perfect!

**Review by awaelchli — APPROVED** [review]
Thanks @rasbt 

I remember adding this line manually a few times 😄

**Pull request merged**